packages/dom-ready/src/index.js

```diff
@@ -14,16 +14,14 @@
  *
  * @return {void}
  */
-const domReady = function( callback ) {
+export default function domReady( callback ) {
 	if (
 		document.readyState === 'complete' || // DOMContentLoaded + Images/Styles/etc loaded, so we call directly.
 		document.readyState === 'interactive' // DOMContentLoaded fires at this point, so we call directly.
 	) {
-		return callback();
+		return void callback();
```

Comment:
Subtle behavioral change - this function did not always return `void` as its type suggested. In this branch, tt would return whatever the callback returned.

Adding the `void` here so the function always returns `undefined`.

Reply:
> Subtle behavioral change - this function did not always return `void` as its type suggested. In this branch, tt would return whatever the callback returned.
> 
> Adding the `void` here so the function always returns `undefined`.

I think this syntax can be a little unexpected for some people, or at least it was for me the first times I had seen it, but I think it's a nice simple way to guarantee the return type as `undefined`, as you suggest.